src/PrimaryAuthController.js

```diff
@@ -107,6 +107,12 @@ function (Okta, PrimaryAuthForm, CustomButtons, FooterRegistration, PrimaryAuthM
       'focusout input': function (e) {
         $(e.target.parentElement).removeClass('focused-input');
       },
+
+      /**
+       * @deprecated
+       * This event was originally added for tracking purpose via MixPanel.
+       * After migrating to new tracking system, it's unnecessary now.
+       */
       'click .button-show': function () {
         this.trigger('passwordRevealed');
```

Comment:
For the event itself, let's use the [deprecate](https://github.com/okta/okta-signin-widget/blob/4e0c5d2823eb6a283f8e6f216eabf7e6eeba1eca/src/util/Logger.js) logger function to ensure the message is passed to a developer. IIRC, these get stripped out of prod builds, so endusers should never see it.